specifyweb/specify/utils.py

```
@@ -18,3 +21,61 @@ def get_app_model(model_name: str):
+
+def create_default_collection_types(apps):
+    Collection = apps.get_model('specify', 'Collection')
+    Collectionobject = apps.get_model('specify', 'Collectionobject')
+    Collectionobjecttype = apps.get_model('specify', 'Collectionobjecttype')
+    code_set = set(Collection.objects.all().values_list('code', flat=True))
+
+    # Create default collection types for each collection, named after the discipline
+    for collection in Collection.objects.all():
+        discipline = collection.discipline
+        discipline_name = discipline.name
+        cot, created = Collectionobjecttype.objects.get_or_create(
```

Ruff (0.15.15)

[warning] Unpacked variable `created` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)